fury/ui.py

```diff
@@ -2319,7 +2356,12 @@ def coord_to_ratio(self, coord):
         ----------
         coord : float
         """
-        return (coord - self.left_x_position) / self.track.width
+        ratio = None
+        if self.orientation == "horizontal":
+            ratio = (coord - self.left_x_position) / self.track.width
```

Comment:
to make Codacy happy, you need to be explicit here. So you can replace `self.track.width` by `float(self.track.width)`